fcrepo-api-x-indexing/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -34,6 +33,8 @@
     <property name="reindexStream" value="${service.reindex.stream}" />
   </bean>
 
+  <bean id="http" class="org.apache.camel.component.http4.HttpComponent" />
```

Comment:
I tend to add this line, too: `<bean id="https" class="org.apache.camel.component.http4.HttpComponent" />`

This allows you to define the reindexing/triplestore URLs as https:// URLs, too.